brightchain-lib/src/lib/blocks/whitened.ts

```
@@ -18,6 +19,15 @@ import { RawDataBlock } from './rawData';
  * 3. Provide privacy without encryption overhead
  */
 export class WhitenedBlock extends RawDataBlock {
+  protected static override checksumService: ChecksumService;
+
+  protected static override initialize() {
+    super.initialize();
```

Biome (1.9.4)

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)